src/components/Input/QtySelector/QtySelector.js

```diff
@@ -105,7 +113,7 @@ class QtySelector extends Component {
           disabled={disabled}
           style={{
             ...style,
-            top: `-${value * QtySelector.INPUT_HEIGHT}px`
+            top: `-${(value % 100) * QtySelector.INPUT_HEIGHT}px`
```

Comment:
Seems like there is an issue on Safari

![Uploading Screen Shot 2019-06-14 at 9.31.23 AM.png…]()